manim/scene/scene_file_writer.py

```diff
@@ -494,6 +494,16 @@ def combine_movie_files(self):
             '-c', 'copy',
             movie_file_path
         ]
+        if not self.save_as_gif:
+            commands+=[
+                "-c", "copy",
+                movie_file_path
+            ]
+        if self.save_as_gif:
```

Comment:
any reason we are not using `else` here?

Reply:
Thanks for pointing that out! It was supposed to be a completely different statement...
Not sure how I missed it, and pretty embarrassing, but the most recent commit fixes it.

Reply:
Nothing embarrassing about missing one statement in a codebase with 20k lines of code ;)